<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CamCapture</class>
 <widget class="QWidget" name="CamCapture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CamCapture</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="pLabelCamView">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>camtest.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pPBCamPlay">
     <property name="text">
      <string>camera start</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pPBSnapShot">
     <property name="text">
      <string>snap shot</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
